main_routers/characters_router.py

```
@@ -0,0 +1,1127 @@
+
+@router.put('/catgirl/{name}')
+async def update_catgirl(name: str, request: Request):
+    data = await request.json()
+    if not data:
+        return JSONResponse({'success': False, 'error': '无数据'}, status_code=400)
+    _config_manager = get_config_manager()
+    characters = _config_manager.load_characters()
+    if name not in characters.get('猫娘', {}):
+        return JSONResponse({'success': False, 'error': '猫娘不存在'}, status_code=404)
+    
+    # 记录更新前的voice_id，用于检测是否变更
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)